src/Microsoft.OpenApi.OData.Reader/Operation/EntitySetGetOperationHandler.cs

```diff
@@ -192,7 +192,8 @@ protected override void SetResponses(OpenApiOperation operation)
                                     }
                                 }
                             }
-                        }
+                        },
+                        Links = Context.CreateLinks(EntitySet)
```

Comment:
Do we have the similar for entity set, singleton get, navigation get?

Reply:
Wouldn't Singleton GET and Navigation GET be handled by the EntityGetOperationHandler?  If not, then yes we would need to add those too.

Reply:
No, Singleton has its own SingletonGet handler. Same for Navigation property.

Reply:
Links have been updated for EntitySet, Singleton and Nagivation (GET)